### PR TITLE
[NETBEANS-1074] Module Review websvc.wsitconf

### PR DESCRIPTION
**Pull request by vikasprabhakar**

### CODE REVIEW

File: enterprise/websvc.wsitconf/licenseinfo.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        <license ref="Apache-2.0-ASF"/>
+        <comment type="COMMENT_UNSUPPORTED"/>
+    </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/websvc/wsitconf/resources/client-keystore.jks</file>
+        <file>src/org/netbeans/modules/websvc/wsitconf/resources/client-truststore.jks</file>
+        <file>src/org/netbeans/modules/websvc/wsitconf/resources/server-keystore.jks</file>
+        <file>src/org/netbeans/modules/websvc/wsitconf/resources/server-truststore.jks</file>
+        <license ref="CDDL-1.0"/>
+        <comment type="CATEGORY_B"/>
```
Comment (matthiasblaesing) [inline enterprise/websvc.wsitconf/licenseinfo.xml]: CATEGORY_B is mapped to the text: "The contained code is used at runtime and must be included in source form.". That does not match this case.  Please remove the type and replace it with: "<comment>The files are sample files taken from Netbeans 8.2 and don't support comments.</comment>"

File: enterprise/websvc.wsitconf/licenseinfo.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  <licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/websvc/wsitconf/resources/templates/DerivedKeyPasswordValidator.template</file>
+        <file>src/org/netbeans/modules/websvc/wsitconf/resources/templates/Saml11HOKCallbackHandler.template</file>
+        <file>src/org/netbeans/modules/websvc/wsitconf/resources/templates/Saml11SVCallbackHandler.template</file>
+        <file>src/org/netbeans/modules/websvc/wsitconf/resources/templates/Saml20HOKCallbackHandler.template</file>
+        <file>src/org/netbeans/modules/websvc/wsitconf/resources/templates/Saml20SVCallbackHandler.template</file>
+        <file>src/org/netbeans/modules/websvc/wsitconf/resources/templates/sts_schema.template</file>
+        <license ref="Apache-2.0-ASF"/>
+        <comment type="COMMENT_UNSUPPORTED"/>
```
Comment (matthiasblaesing) [inline enterprise/websvc.wsitconf/licenseinfo.xml]: This is not correct - comments are actually supported, but at this point we choose to allow templates without ASF headers. So please use type='TEMPLATE_MINIMAL_IP'.

I noticed, that `sts.template` and `sts13.template` both contain an ASF header, while the other files don't. I would remove the headers from these two files and add these to this fileset, to unify that.

**Comment by vikasprabhakar** [comment]
@matthiasblaesing please review the changes

**Comment by matthiasblaesing** [comment]
Thank you I pushed a slightly modified changeset as:

https://github.com/apache/incubator-netbeans/commit/a8dbf8b18e01f7f73ff20879a897e7606e7c4d30

The problem was here:

https://github.com/apache/incubator-netbeans/blob/04cabc2740cc6e68f793583309d47cd9c12a27d9/enterprise/websvc.wsitconf/licenseinfo.xml#L39

The type is an enum - in this case you supply a "free-text" comment. That comment needs to supplied as content of the element. See difference here:

https://github.com/apache/incubator-netbeans/blob/a8dbf8b18e01f7f73ff20879a897e7606e7c4d30/enterprise/websvc.wsitconf/licenseinfo.xml#L39

**Pull request closed without merging**